rust/crd/src/constants.rs

```diff
@@ -40,10 +42,10 @@ pub const MIN_MEMORY_OVERHEAD: u32 = 384;
 pub const JVM_OVERHEAD_FACTOR: f32 = 0.1;
 pub const NON_JVM_OVERHEAD_FACTOR: f32 = 0.4;
 
-pub const MAX_SPARK_LOG_FILES_SIZE_IN_MIB: u32 = 10;
-pub const MAX_INIT_CONTAINER_LOG_FILES_SIZE_IN_MIB: u32 = 1;
```

Comment:
The log file size for the init container should still be listed.

Reply:
I think this is not needed.

`SparkApplications` have up to three init containers but none of them use log4j for logging:
* `job` - copies the user jar-file to a dedicated place
* `requirements` - installs Python pip packages
* `tls` - generates a trust store

The spark history server has no init (or prepare) containers.

Reply:
`capture_shell_output` is used in the init containers, so log files are written even if they contain only few kilobytes.

Reply:
hm ok, wan't clear about that. Then I'll set it to 3Mib for consistency

Reply:
Done https://github.com/stackabletech/spark-k8s-operator/pull/259/commits/19c92641f8b6e61e7de6dcca602d15492ad6d317